docs/why-qlik-core.md

```diff
@@ -0,0 +1,52 @@
+# An analytics development platform by Qlik
+
+Qlik Core is built around the Qlik Associative Engine and Qlik-authored open source libraries.
+Who’s Qlik you might ask? We’ve been around for 25 years and our specialty is analytics.
+Our platform has powerful features like responsive and intelligent visualizations,
+collaboration and storytelling functionality, robust data control and role-based data security.
+Underneath those features is our "secret sauce" that we call the Qlik Associative Engine.
+This in-memory inference and calculation engine is what’s responsible for the magic —
+dynamically indexing and associating data without needing to rely on query-based analyses
+which are restricted to linear exploration within a partial view of the data.
+
+With Qlik Core, you can take advantage of this powerful associative engine to build,
+extend and quickly deploy custom interactive data-driven solutions that are highly scalable
+and cloud-ready. Supported by a backbone of Linux and Docker, Qlik Core applications can
```

Comment:
backbone of Linux and docker sounds weird to me.
Change to something like "Produced as docker images, Qlik Core applications can.."